playbooks/provisioning/openstack/README.md

```diff
@@ -19,6 +19,20 @@ There are no additional dependencies for the cluster nodes. Required
 configuration steps are done by Heat given a specific user data config
 that normally should not be changed.
 
+## Required galaxy modules
+
+In order to pull in external dependencies for DNS configuration steps,
+the following commads need to be executed:
+
+    ansible-galaxy install \
+      -r playbooks/provisioning/openstack/galaxy-requirements.yaml \
+      -p openshift-ansible-contrib/playbooks/provisioning/openstack/roles
```

Comment:
the guide assumes we're cd-ed into the directory that contains openshift-ansible-contrib, and inventory in the root path